kcidev/subcommands/bisect.py

```diff
@@ -133,6 +137,20 @@ def init_bisect(state):
     return commitid
 
 
+def update_tree(workdir, branch, giturl):
+    if not os.path.exists(workdir):
+        click.secho(
+            "Cloning repository (this might take significant time!)", fg="green"
+        )
+        repo = Repo.clone_from(giturl, workdir)
+        repo.git.checkout(branch)
+    else:
+        click.secho("Pulling repository", fg="green")
```

Comment:
same here, this would go to stdout potentially messing with parseable content from stdout.